src/main/java/software/amazon/awssdk/crt/CRT.java

```diff
@@ -251,23 +252,30 @@ private static void jvmInit() {
     public static native String awsErrorName(int errorCode);
 
     /**
-     * @return The number of bytes allocated in native resources. If aws.crt.memory.tracing is 1 or 2, this will
-     *         be a non-zero value. Otherwise, no tracing will be done, and the value will always be 0
+     * @return The number of bytes allocated in native resources. If
+     *         aws.crt.memory.tracing is 1 or 2, this will be a non-zero value.
+     *         Otherwise, no tracing will be done, and the value will always be 0
      */
     public static long nativeMemory() {
         return awsNativeMemory();
     }
 
-    private static native long awsNativeMemory();
+    /**
+     * Memory Tracer dump when trace level is 2
+     */
+    public static native void awsMemTracerDump();
```

Comment:
I just added this function in this file

Reply:
The dump isn't visible unless logging is enabled at Trace level. Tell users that.

Also remind them how to set the correct system properties, don't assume they've read it in the docs for some other function

Also make the name consistent with the existing public "nativeMemory()" function

```
/**
 * Dump info to logs about all memory currently allocated by native resources.
 * The following system properties must be set to see a dump:
 * aws.crt.memory.tracing must be 1 or 2
 * aws.crt.log.level must be "Trace"
 */
public static void dumpNativeMemory()